### PR TITLE
ClassicUI: Fixes dimmer/rolershutter infinite request bug (Bugzilla 460320)

### PR DESCRIPTION
**Pull request by sja**
This belongs to Bugzilla issue https://bugs.eclipse.org/bugs/show_bug.cgi?id=460320.

On long presses, the dimmer widget does XHRs forever, because the touchend-Handler wasn't attached correctly. The attribute was `touchend="..."` which has to be `ontouchend="..."`.

Additional goodies:
* Add quotes to all HTML arguments
* Introduced document.OH namespace to encapsulate openHAB stuff and don't pollute the global namespace
* Remove evil eval in setTimeout
* Removed unused parameters
* Grouped dimmer and rollershutter in own state holding objects
* On page change, reset all timers/intervals to be sure not to fire. (Special use case for multitouch)

### CODE REVIEW

**Comment by sja** [comment]
I re-added Firefox.css and SmartHome.css, they were gone, which produces a 404, which is bad.

File: bundles/ui/org.eclipse.smarthome.ui.classic/snippets/rollerblind.html [hunk 1]
```diff
@@ -1,2 +1,2 @@
...
+<li style="%labelstyle%"><img src="/images/%icon%.png" width="29" height="29" class="iFull" /><span class="options"><img src="images/_down.png" ontouchstart="ESH.buttonPressed('../CMD?%item%=DOWN');event.stopPropagation();event.preventDefault();" ontouchend="ESH.buttonReleased('../CMD?%item%=DOWN', '../CMD?%item%=STOP');event.stopPropagation();event.preventDefault();" onMouseDown="ESH.buttonPressed('../CMD?%item%=DOWN')" onMouseUp="ESH.buttonReleased('../CMD?%item%=DOWN', '../CMD?%item%=STOP')" /><img src="images/_stop.png" onclick="ESH.changeState('../CMD?%item%=STOP')" /><img src="images/_up.png" ontouchstart="ESH.buttonPressed('../CMD?%item%=UP');event.stopPropagation();event.preventDefault();" ontouchend="ESH.buttonReleased('../CMD?%item%=UP', '../CMD?%item%=STOP');event.stopPropagation();event.preventDefault();" onMouseDown="ESH.buttonPressed('../CMD?%item%=UP')" onMouseUp="ESH.buttonReleased('../CMD?%item%=UP', '../CMD?%item%=STOP')" /></span style="%valuestyle%">%label%</li>
```
Comment (kaikreuzer) [inline bundles/ui/org.eclipse.smarthome.ui.classic/snippets/rollerblind.html]: Did you put the absolute path ("/") back on purpose? Please note that these had been removed with https://github.com/eclipse/smarthome/commit/3a5e695fd8fa5b469c17a5f4aba25157f2be61da in order to make it possible to use the UI with reverse proxies on different urls.

**Comment by kaikreuzer** [comment]
> I re-added Firefox.css

I assume you have checked the copyright and authorship of this file and that you can contribute it according to your CLA.

**Comment by kaikreuzer** [comment]
Thanks for this contribution. As it includes a lot of changes, I will have to create a CQ for the IP team, but I will first wait for your answer to my question above.

**Comment by sja** [comment]
The File is part of WebApp.Net <http://webapp.net/> (trywebapp.net <http://trywebapp.net/>) which is licensed under new BSD. Cite: "WebApp.Net is an open source project released under new BSD license."
Is that enough?

**Comment by kaikreuzer** [comment]
> Is that enough?

Ah, ok, I didn't know that it was part of the WebApp.Net itself. I wonder why the file went missing then, because I took the sources from there. So yes, this information is enough.

**Comment by sja** [comment]
I updated the relative URLs.

**Comment by kaikreuzer** [comment]
Thanks, I've created CQ 9307.

**Comment by kaikreuzer** [comment]
It appears as if these fixes introduce some new bugs, see https://github.com/openhab/openhab/issues/2210 and https://github.com/openhab/openhab/issues/2214.
Let's create a new PR, once these are solved.

**Pull request closed without merging**